operator/operator.make

```diff
@@ -65,11 +46,10 @@ PATCH_REPLACES := sed -i -e 's;\(.*\)\(version:.*\);\1replaces: pmem-csi-operato
 PATCH_DATE := sed -i -e 's;\(.*createdAt: \).*;\1$(shell date +%FT%TZ);g'
 
 # Generate CRD
-operator-generate-crd: controller-gen
+operator-generate-crd: $(CONTROLLER_GEN)
 	@echo "Generating CRD in $(CRD_DIR) ..."
-	cd $(REPO_ROOT) && $(CONTROLLER_GEN) crd:trivialVersions=true,crdVersions=v1 paths=./pkg/apis/... output:dir=$(CRD_DIR)/
-	version="$(shell $(CONTROLLER_GEN) --version |cut -f2 -d':')"; \
-    sed -i "1s/^/# This file was generated by controller-gen$$version via 'make operator-generate-crd'\n/" $(CRD_DIR)/*
+	$(CONTROLLER_GEN) crd paths=./pkg/apis/... output:dir=$(CRD_DIR)
```

Comment:
`trivialVersions` was not recognized as parameter anymore. Do you know what it did?

`crdVersions=v1` should be the default now.